guides/implementation/central_mssql.rst

```diff
@@ -99,4 +93,4 @@ Verify the service is running smoothly:
   systemctl restart mmcentral
   systemctl status mmcentral
 
-If there are any database connection issues, the service will fail to start. Check the Central startup log for errors, located in the Central data directory at either ``/var/mmsuite/mmcentral/logs`` or ``C:\ProgramData\Men and Mice\Central\logs``.
+If there are any database connection issues, the service will fail to start. Check the Central startup log for errors, located in the Central data directory at either ``/var/micetrosuite/mmcentral/logs`` or ``C:\ProgramData\Micetro\Central\logs``.
```

Comment:
file path locations are still /var/mmsuite and C:\ProgramData\Men and Mice\ 

But the CreateDatabase script is being updated in the DevOps PR here https://dev.azure.com/menandmice/_git/Suite/pullrequest/9761, where I decided to use the opportunity and change the name of the database objects we create to micetro. Changing the filepaths on Micetro servers is whole another (and bigger) story